noetic/distribution.yaml

```diff
@@ -1242,6 +1242,24 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  darknet_ros:
+    doc:
+      type: git
+      url: https://github.com/leggedrobotics/darknet_ros.git
+      version: master
+    release:
+      packages:
+      - darknet_ros_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/leggedrobotics/darknet_ros-release.git
+      version: 1.1.5-1
+    source:
+      test_pull_requests: true
```

Comment:
@mbjelonic have you [setup your repository](http://wiki.ros.org/buildfarm/Pull%20request%20testing) for this?

Reply:
Yes, I set this up already before since this is not my first time, I pushed darknet_ros to rosdistro. You can find the previous version here:

https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml